src/applications/check-in/actions/day-of/index.js

```diff
@@ -1,5 +1,14 @@
 import { updateForm } from '../../utils/navigation/day-of';
 
+export const RECORD_ANSWER = 'RECORD_ANSWER';
+
+export const recordAnswer = answer => {
```

Comment:
I wonder if we could re-use some of the actions from the pre-check-in form answers

Reply:
Definitely some room for re-use. Would you be okay with me adding a separate tech debt ticket for that? The main reason would just be to keep the PR size down for this one

Reply:
Completely cool with that being a seperate ticket

Reply:
Added actions clean up ticket to backlog